ddtrace/context.py

```diff
@@ -22,6 +22,9 @@ class Context(object):
 
     This data structure is thread-safe.
     """
+    _partial_flush_enabled = asbool(get_env('tracer', 'partial_flush_enabled', 'false'))
```

Comment:
:+1: let's remember to add a configuration option for this when we get around to consolidating the tracer configuration.